src/xrCore/LocatorAPI.cpp

```diff
@@ -36,7 +36,7 @@ static constexpr pcstr FSLTX = "fs.ltx"
 static constexpr pcstr FSLTX = "fsgame.ltx";
 #endif
 
-    struct _open_file
+struct _open_file
```

Comment:
And again we see the mix 'styling' changes and real corrections of the code. Don't do that.